build_tools/github_actions/ci_build_bazel.sh

```diff
@@ -22,5 +22,5 @@ if [[ $# -ne 0 ]] ; then
 fi
 
 # Build and Test StableHLO
-bazel build --lockfile_mode=error //...
-bazel test //...
+bazel build --lockfile_mode=error //... --config=asan --config=ubsan
```

Comment:
Consider creating a separate config that wraps these two, e.g. `build:san --config=asan,ubsan` and then you can do `--config=san` here.